### DISCUSSION TITLE
Issue with focus on new item when useFieldArray prepend.

### DISCUSSION THREAD

**Discussion by hmtoan96** [react-hook-form/react-hook-form#9138] [Q&A]
Hello everyone!

I have an issue is when I prepend or append new value with useFieldArray, it'll focus on new input but I don't want behavior like this.

Can I remove focus behavior?

Thank for your support!

**Comment by bluebill1049** [comment]
https://react-hook-form.com/api/usefieldarray/

![Screen Shot 2022-10-04 at 7 49 33 pm](https://user-images.githubusercontent.com/10513364/193776567-815a9008-e581-4bb7-84c9-a5b5dd8371f5.png)

`focusOptions`

**Reply by hmtoan96** [reply]
Thank Bill for the answer! I already use focusName in focusOptions, make it focus on another hidden input and it work perfectly. After that I just realize that it have shouldFocus option. Btw thank you for your time <3

**Reply by bluebill1049** [reply]
```tsx
{
  shouldFocus: false
}
```
